README.md

```diff
@@ -487,6 +487,7 @@ Services to securely store your Docker images.
 
 # Useful Resources
 * [Awesome Linux Container](https://github.com/Friz-zy/awesome-linux-containers) more general about container than this repo, by [@Friz-zy](https://github.com/Friz-zy).
+* [Awesome Sysadmin](https://github.com/n1trux/awesome-sysadmin) by [@n1trux](https://github.com/n1trux)
```

Comment:
Cool, thanks for sharing @Moshe-Immerman

Just a small belated remark: It would be nice adding a description like in the previous line 489.
Maybe @veggiemonk can adjust this on PR #369